core/src/main/java/org/elasticsearch/action/admin/indices/close/TransportCloseIndexAction.java

```diff
@@ -99,4 +104,13 @@ public void onFailure(Throwable t) {
             }
         });
     }
+
+    @Override
+    public void onRefreshSettings(Settings settings) {
+        final boolean enable = settings.getAsBoolean(SETTING_CLUSTER_INDICES_CLOSE_ENABLE, this.closeIndexEnabled);
```

Comment:
Should this be 
```java
+        final boolean enable = settings.getAsBoolean(SETTING_CLUSTER_INDICES_CLOSE_ENABLE, true);
```
?

Settings can't really be cleared right now but if they could be this would stick the value to whatever it was set at before clearing. Maybe it doesn't matter but I'm certainly more used to seeing the default value here.